src/main/java/model/item/Jewellery.java

```diff
@@ -20,6 +22,14 @@ public Jewellery copy() {
         return jewellery;
     }
 
+    public String getGem() {
+        return gem;
```

Comment:
what are gems used for?

Reply:
They have no particular use, except for filtering.